packages/api/src/server.js

```diff
@@ -39,6 +40,7 @@ let fastify
 
 module.exports = {
   start: async () => {
+    await Constants.genesisTime
```

Comment:
Why this is needed? I thought it would lazy load in the places where it is necessary